src/index.html

```diff
@@ -7,8 +7,69 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+
+        <a class="logo__link" href="#home">
+          <img
+            src="images/logo.png"
+            alt="Logo"
+            class="logo"
+          >
+        </a>
+
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a
+            class="nav__link is-active"
+            href="#apple">Apple
+            </a>
+          </li>
```

Comment:
You don't need all of these spaces (if you see/feel it ruins readability – just don't add them)